test/functionalTest/cases/1156_mq_as_uri_param_for_metadata/q_and_mq_as_uri_param_for_metadata.test

```diff
@@ -0,0 +1,454 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+mq as URI param for metadata in filters
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. Create entity E1, with attr A1=1, having metadata M1=1
+# 02. Create entity E2, with attr A1=1, having metadata M1=2
+# 03. Create entity E3, with attr A1=1, having metadata M2=3
+# 04. GET entities matching q=A1==1 AND mq=A1.M1==2, see E2
```

Comment:
The `q` uses in all the cases will always match all entities. Maybe you should create some additional entities with A1 different from 1 and with A2 attribute instead of A1 in to check that the test is not capturing them.